### ISSUE TITLE
A11y_SQL Azure Data Studio_Add an Azure account_Delete an Azure account_Screenreader:Screenreader announce all the information in one go.

### ISSUE THREAD

**Issue by SwatiTiwari-TCS** [microsoft/azuredatastudio#15838]
Check out Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier."

GitHubTags: #A11yMAS;#A11yTCS;#A11ySev2;#BM_SQLAzureDataStudio_Win32_Jun2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#WCAG1.3.1;#NVDA;

### Environment Details:
Application Name: SQL Azure Data Studio
Version: 1.30.0-insider (user setup)
Windows Version: Win10
Screen reader : JAWS, NVDA2020.2

### Repro Steps:

1. Launch the application-Azure Data Studio.
2. Select the Connections icon (if not already selected) from left bar.
3. Add an Azure account using the steps in Benchmark Scenario #5 listed above.
4. Once the account is successfully added, hover over the 'Azure' header in the Connections pane and select the '+' icon again. A right blade should appear.
5. Select 'X' on the account you would like to delete.
6. Select 'Yes' on the warning pop-up that appears.
7. Validate that the account has been deleted.
8. Start screen reader and observe its announce all the information in one go after opening of delete dialog.

### Actual:
Screen reader announce All the information in one go after opening of delete dialog.

**This issue is observed with NVDA only.**

### Expected:
Screen reader should not announce the all the information in one go as screen reader user will face difficulty.

### User Impact:
Screenreader only user will face difficulty to understand .

**Comment by SwatiTiwari-TCS** [comment]
Issue is not reproing, Hence closing the bug.

**Comment by Harika1702** [comment]
GitHub Tags: #Closed;
